modin/pandas/io.py

```diff
@@ -53,7 +53,7 @@ def _read(**kwargs):
     -------
     modin.pandas.DataFrame
     """
-    from modin.data_management.factories.dispatcher import FactoryDispatcher
+    from modin.core.execution.dispatching.factories.dispatcher import FactoryDispatcher
```

Comment:
same about moving to the top - can we do that?

Reply:
Even if we can, this should go in a separate issue/PR